Assets/_BForBoss/_Weapons/Scripts/Weapons/WeaponData.cs

```diff
@@ -0,0 +1,36 @@
+namespace Perigon.Weapons
+{
+    public readonly struct WeaponData
+    {
+        public readonly float ElapsedRateOfFire;
+        public readonly float ElapsedReloadDuration;
+        public readonly int ElapsedAmmunitionAmount;
+        public readonly bool IsReloading;
```

Comment:
Alot of these values will be gone soon. But Moving them over to make sure everything still works